### ISSUE TITLE
Track Accessibility->Text Size on Windows instead of Segoue UI 12 Default

### ISSUE THREAD

**Issue by christophwille** [AvaloniaUI/Avalonia#21966]
### Is your feature request related to a problem? Please describe.

I will preface this that our clanker didn't find an in-built feature in Avalonia, and our UI doesn't honor Settings / Accessibility / Text size by default, so I think this is a real missing feature.

I have the following setting:

<img width="1016" height="106" alt="Image" src="https://github.com/user-attachments/assets/641843dc-97aa-435f-a33a-e288095836cf" />

It is honored by Windows/WPF apps, but Avalonia seemingly doesn't pick it up. Please see https://github.com/icsharpcode/ILSpy/pull/3987 (WIP) what our clanker put together.

### Describe the solution you'd like

It would be great if it would "simply work", ie Avalonia picking up that Windows settings and adapting the default font size.

### Describe alternatives you've considered

Currently experimenting with our PR.

**Comment by MrJul** [comment]
#20292 implements this (we still need to agree on the exact API shape, but we want that feature in the core framework).
